### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ServiceSettings_AI4D:The name of the edit fields is not set according to its visual label.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9292]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yWCAG2.1;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS2.5.3;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Tools:Accessibility insights for desktop
MAS References:MAS2.5.3

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Select Configuration profile --> Update Azure settings --> Update Cluster settings & hit Next.
8. Launch accessibility insights for desktop and hover over the edit field under "Endpoint settings" button.

**Actual:**
On hovering over all the edit boxes under the "Endpoint settings" button. the name for it is set as "Controller port". While the label is set as "Controller "
This issue is for all the edit fields under the button.

**Expected:**
The name should match the label of the edit box . The label should be "Controller port" 
Screen readers should also announce the name that has been set.

**User Impact:**
Speech to text users will face difficulty in accessing the edit box since the name present on the screen of the label won't match the edit box.

**Attachment for Reference :**
![image](https://user-images.githubusercontent.com/24580570/75134585-bff1c500-5704-11ea-9fae-6e6e0ddcc254.png)

**Comment by alanrenmsft** [comment]
now I've made it always show the port text so that user knows it is a port field

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_Win32;#A11yMAS;

**Comment by sspatil123** [comment]
As per PO comment big data cluster is removed. Hence closing bug. GithubTags:#Closed;Version 1.42.0

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
